coralproject/talk
# [next] Fix Build Issues

## What does this PR do?

Replaces `npm-run-all` with `@coralproject/npm-run-all` until https://github.com/mysticatea/npm-run-all/pull/171 is merged and released.

## How do I test this PR?

```bash
export NODE_OPTIONS=--max-old-space-size=100
npm run build
echo $?
```

You should see a number that is non-zero.
